wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -549,6 +553,58 @@ public static function setOfficeDetails($op, $groupID, $entityID, &$params) {
 
   }
 
+  /**
+   * This hook is called after the database write on a custom table.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The custom group ID.
+   * @param int $objectId
+   *   The entityID of the row in the custom table.
+   * @param object $objectRef
+   *   The parameters that were sent into the calling function.
+   */
+  public static function displayActivityDataForCollectionCamp($op, $groupID, $entityID, &$params) {
```

Comment:
Also I don't think this is correct naming convention:
```suggestion
  public static function displayActivityDataForCollectionCamp($op, $groupId, $entityId, &$params) {
```

Reply:
The naming convention is correct i was able to get data on `$groupID, $entityID` not on `$groupId, $entityId,` by adding error logs